src/octopeer-github/content/EventTracker/WindowResolutionTracker.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Created by Mathias on 2016-05-26.
+ */
+class WindowResolutionTracker {
+
+    /** Private static final for the timeout between logs. */
+    private static get WINDOW_RESOLUTION_TRACKER_TIMEOUT() { return 500; }
+
+    /**
+     * Initialize a WindowResolutionTracker that contains a WindowResolutionDatabaseAdaptable.
+     * @param db The DatabaseAdaptable for the Tracker.
+     */
+    constructor(private db: WindowResolutionDatabaseAdaptable) {
+        const self = this;
+        let $window: JQuery = $(window);
+        let resizeTimer: number;
+        $window.resize(function (eventObject: JQueryEventObject) {
+            clearTimeout(resizeTimer);
+            resizeTimer = setTimeout(function () {
+                self.db.postWindowResolution(new WindowResolutionEvent($window.width(), $window.height(),
+                    new Date().getTime()), EMPTY_CALLBACK, EMPTY_CALLBACK);
+            }, WindowResolutionTracker.WINDOW_RESOLUTION_TRACKER_TIMEOUT);
```

Comment:
I am not going to be able to fix this before 7, unfortunately. I'll just finish the tests and then it's done.